### PR TITLE
Adding some classes for new secretsFindins list

### PR DESCRIPTION
**Pull request by dor-hayun**

### CODE REVIEW

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/AgentProjectInfo.java [hunk 1]
```diff
@@ -141,4 +142,8 @@ public String getViaLanguageName() {
     public void setViaLanguageName(String viaLanguageName) {
         this.viaLanguageName = viaLanguageName;
     }
+
+    public Collection<SecretFinding> getSecretFindings() {
```
Comment (samisalamiws) [inline wss-agent-api/src/main/java/org/whitesource/agent/api/model/AgentProjectInfo.java:146]: add the `setSecretFindings()` method as well

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/Code.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package org.whitesource.agent.api.model;
+
+import java.util.List;
+
+public class Code {
+    private List<Line> lines;
```
Comment (samisalamiws) [inline wss-agent-api/src/main/java/org/whitesource/agent/api/model/Code.java:6]: add setters and getters for all fields

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/Line.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package org.whitesource.agent.api.model;
+
+
+public class Line {
+    private int number;
```
Comment (samisalamiws) [inline wss-agent-api/src/main/java/org/whitesource/agent/api/model/Line.java:5]: add setters and getters for all fields

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/SecretFinding.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package org.whitesource.agent.api.model;
+
+
+public class SecretFinding {
+
+    private int layerNumber;
```
Comment (samisalamiws) [inline wss-agent-api/src/main/java/org/whitesource/agent/api/model/SecretFinding.java:6]: add setters and getters for all fields

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/AgentProjectInfo.java (hunk 1 above)
Comment (dor-hayun) [reply wss-agent-api/src/main/java/org/whitesource/agent/api/model/AgentProjectInfo.java:146]: done

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/Code.java (hunk 2 above)
Comment (dor-hayun) [reply wss-agent-api/src/main/java/org/whitesource/agent/api/model/Code.java:6]: done

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/Line.java (hunk 3 above)
Comment (dor-hayun) [reply wss-agent-api/src/main/java/org/whitesource/agent/api/model/Line.java:5]: done

File: wss-agent-api/src/main/java/org/whitesource/agent/api/model/SecretFinding.java (hunk 4 above)
Comment (dor-hayun) [reply wss-agent-api/src/main/java/org/whitesource/agent/api/model/SecretFinding.java:6]: done

**Review by samisalamiws — APPROVED** [review]

**Pull request merged**